packages/core/src/test-runner/TestRunnerPool.ts

```diff
@@ -58,8 +58,8 @@ export class TestRunnerPool implements Disposable {
       numConcurrentRunners--;
```

Comment:
Wow, we still have a `TestRunnerPool`? This class is replaced by the generic `Pool<T>` 😅. Will remove in another PR.